### PR TITLE
fix(timetable): apply change requests

### PR DESCRIPTION
**Pull request by osminaz**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

-

## Browsers

I tested the build on the following browsers:

- [ ] Firefox Desktop
- [ ] Chrome Desktop
- [ ] Edge Desktop
- [ ] Safari Desktop
- [ ] Chrome Mobile
- [ ] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/components/sbb-pearl-chain/sbb-pearl-chain.sample-data.js [hunk 1]
```diff
@@ -40,7 +38,7 @@ const unplannedStopService = {
 
 const redirectedService = {
   serviceAlteration: {
-    redirected: true,
+    redirectedFormatted: 'Ausnahmsweise kein Halt',
```
Comment (jeripeierSBB) [inline src/components/sbb-pearl-chain/sbb-pearl-chain.sample-data.js:41]: Prefer English, but as always you can also leave it as it is.

File: src/global/helpers/date-helper.ts [hunk 2]
```diff
@@ -34,3 +42,35 @@ export function removeTimezoneFromISOTimeString(isoTime: string): Date | undefin
...
+ * */
+
+export const durationToTime = (duration: number): string => {
+  const result = [];
+  const now = 0;
+  let future = addMinutes(now, duration);
+
+  const days = differenceInDays(future, now);
+  if (days > 0) {
+    result.push(`${days} d`);
```
Comment (jeripeierSBB) [inline src/global/helpers/date-helper.ts:60]: Is this really intended, that there is no i18n now?

File: src/global/helpers/date-helper.ts [hunk 3]
```diff
@@ -34,3 +42,35 @@ export function removeTimezoneFromISOTimeString(isoTime: string): Date | undefin
...
+    future = subDays(future, days);
+  }
+
+  const hours = differenceInHours(future, now);
+  if (hours > 0) {
+    result.push(`${hours} h`);
```
Comment (jeripeierSBB) [inline src/global/helpers/date-helper.ts:66]: Is this really intended, that there is no i18n now?

File: src/global/helpers/date-helper.ts [hunk 4]
```diff
@@ -34,3 +42,35 @@ export function removeTimezoneFromISOTimeString(isoTime: string): Date | undefin
...
+    future = subHours(future, hours);
+  }
+
+  const minutes = differenceInMinutes(future, now);
+  if (minutes > 0) {
+    result.push(`${minutes} min`);
```
Comment (jeripeierSBB) [inline src/global/helpers/date-helper.ts:72]: Is this really intended, that there is no i18n now?

File: src/global/helpers/date-helper.ts (hunk 2 above)
Comment (osminaz) [reply src/global/helpers/date-helper.ts:60]: Yes, sbb mobile solve this like that aswell.

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**